### ISSUE TITLE
'cargo report future-incompatibilities' error

### ISSUE THREAD

**Issue by taufik-rama** [harporoeder/tokio-nsq#11]
Hello :wave: 

Seems one of the dependency uses some will-be-rejected code

```
taufik-rama@box:~/programming/rust/taufik-rama/tokio-nsq$ cargo build
    Updating crates.io index
    # ...
    Finished dev [unoptimized + debuginfo] target(s) in 27.44s
warning: the following packages contain code that will be rejected by a future version of Rust: cargo-lock v4.0.1
note: to see what the problems were, use the option `--future-incompat-report`, or run `cargo report future-incompatibilities --id 1`
```

```
taufik-rama@box:~/programming/rust/taufik-rama/tokio-nsq$ rustc --version
rustc 1.70.0 (90c541806 2023-05-31)
```

It comes from `built` build-dep, bumping the version to `0.6` seems to fix the issue

**Comment by harporoeder** [comment]
Fixed in release '0.13.1.

**Comment by taufik-rama** [comment]
Thanks!
